src/main/scala/io/opentargets/etl/backend/literature/Processing.scala

```diff
@@ -146,17 +146,15 @@ object Processing extends Serializable with LazyLogging {
       "literatureIndex" -> IOResource(literatureIndexAlt._1, outputs.literatureIndex),
       "publicationSentences" -> IOResource(literatureIndexAlt._2, outputs.literatureSentences)
     ) ++ {
-      if (context.configuration.literature.processing.writeFailures) {
+      if (empcConfiguration.writeFailures) {
```

Comment:
I can't find this refactoring from _context.configuration..._ into _empcConfiguration_
In case it's somewhere outside my radar, there is a typo, the resource is **Europe PMC**, so it should be _epmcConfiguration_.
In any case, whether to write down the failed cooccurrences and/or failed matches is a configuration parameter of the literature pipeline, and not the _Europe PMC_ data ingestion.

Reply:
In line 125 of Processing.scala we're extracting processing configuration `val empcConfiguration: LiteratureProcessing = context.configuration.literature.processing` to this variable not sure what was behind the naming of the variable but can be renamed to processingConfiguraiton